python/lsst/obs/base/ingest.py

```diff
@@ -42,28 +43,52 @@
     Formatter,
 )
 from lsst.pex.config import Config, ChoiceField, Field
-from lsst.pipe.base import Task
+from lsst.pipe.base import Task, timeMethod
 
 from ._instrument import Instrument, makeExposureRecordFromObsInfo
 from ._fitsRawFormatterBase import FitsRawFormatterBase
 
 
 def _do_nothing(*args, **kwargs) -> None:
-    """A function that accepts anything and does nothing, for use as a default
-    in callback arguments.
+    """Do nothing.
+
+    This is a function that accepts anything and does nothing.
+    For use as a default in callback arguments.
     """
     pass
 
 
+def _log_msg_counter(noun: Union[int, Iterable]) -> Tuple[int, str]:
+    """Count the iterable and return the count and plural modifier.
```

Comment:
I'm not saying do it on this ticket, but this is the kind of thing that might be nice to live base or somewhere we can use it more broadly (until someone wants to make it smarter with es plurals...).